webapp/views.py

```diff
@@ -1246,84 +1247,42 @@ def dispatch_request(self):
 
 # Cube
 # ===
+cube_api = CubeAPI(
+    "https://qa.cube.ubuntu.com",
+    os.getenv("CUBE_EDX_CLIENT_ID"),
+    os.getenv("CUBE_EDX_CLIENT_SECRET"),
+    talisker.requests.get_session(),
+)
+
+
 def cube_microcerts():
     user = user_info(flask.session)
 
     if user:
-        data = {
-            "user": {"name": user["fullname"]},
-            "modules": [
-                {
-                    "badge": (
-                        "https://assets.ubuntu.com/v1/"
-                        "75f71a36-placeholder-badge-orange.svg"
-                    ),
-                    "name": "Ubuntu system architecture",
-                    "number": "1",
-                    "topics": [
-                        "Disable a faulty device",
-                        "Disable specific hardware",
-                        "Adjust Kernel module parameters",
-                        "Record the Ubuntu version information",
-                        "Determine and record specific CPU capabilities",
-                        "Create a system information Tarball",
-                        "Create a system information JSON",
-                        "Identify the running Kernel and associated files",
-                    ],
-                    "test_url": "",
-                    "training_url": "",
-                    "status": "Passed",
-                    "action": "",
-                    "date_attempted": "20-12-07",
-                },
-                {
-                    "badge": (
-                        "https://assets.ubuntu.com/v1/"
-                        "e3dd001a-placeholder-badge-white.svg"
-                    ),
-                    "name": "Managing packages in Ubuntu",
-                    "number": "2",
-                    "topics": [
-                        "Install a sepcific Snap",
-                        "Configuring Snap updates",
-                        "Determine Snap confirnement level",
-                        "Identify package dependencies",
-                        "Find and install a pervious package verison",
-                        "Control package upgrades",
-                        "Install from a PPA respository",
-                        "Reconfigure an installed package",
-                        "Change the Binary used for a Utility",
-                        "Configure automatic updates",
-                    ],
-                    "test_url": "http://module2.url",
-                    "training_url": "http://module3.url",
-                    "status": "Failed",
-                    "action": "Retest",
-                    "date_attempted": "20-12-07",
-                },
+        courses = cube_api.get_courses(organization="ubuntu")["results"]
+
+        modules = []
+        for course in courses:
```

Comment:
Would this cause a 500 if `get_courses` had returned an error/something other than an iterable?